tests/python_tests/samples/conftest.py

```diff
@@ -0,0 +1,160 @@
+import subprocess # nosec B404
+import os
+import pytest
+import shutil
+import logging
+import gc
+
+from utils.network import retry_request
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Dictionary containing model configurations.
+# Each key is a model identifier, and the value is a dictionary with:
+# - "name": the model's name or path
+# - "convert_args": a list of arguments for the conversion command
+MODELS = {
+    "LaMini-GPT-124M": { 
+        "name": "MBZUAI/LaMini-GPT-124M",
+        "convert_args": []
+    },
+    "SmolLM-135M": {
+        "name": "HuggingFaceTB/SmolLM-135M",
+        "convert_args": ['--trust-remote-code', '--weight-format', 'fp16']
+    },
+    "SmolLM2-135M": {
+        "name": "HuggingFaceTB/SmolLM2-135M",
+        "convert_args": ['--trust-remote-code']
+    },  
+    "WhisperTiny": {
+        "name": "openai/whisper-tiny",
+        "convert_args": ['--trust-remote-code']
+    },
+    "Qwen2.5-0.5B-Instruct": {
+        "name": "Qwen/Qwen2.5-0.5B-Instruct",
+        "convert_args": ['--trust-remote-code', '--weight-format', 'fp16']
+    },
+    "Qwen2-0.5B-Instruct": {
+        "name": "Qwen/Qwen2-0.5B-Instruct",
+        "convert_args": ['--trust-remote-code', '--weight-format', 'fp16']
+    },
+    "phi-1_5": {
+        "name": "microsoft/phi-1_5",
+        "convert_args": ['--trust-remote-code', '--weight-format', 'fp16']
+    },
+    "TinyStories-1M": {
+        "name": "roneneldan/TinyStories-1M",
+        "convert_args": ['--trust-remote-code', '--weight-format', 'fp16']
+    }
+}
+
+TEST_FILES = {
+    "how_are_you_doing_today.wav": "https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/librispeech_s5/how_are_you_doing_today.wav",
+    "adapter_model.safetensors": "https://huggingface.co/smangrul/tinyllama_lora_sql/resolve/main/adapter_model.safetensors"
+}
+
+SAMPLES_PY_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../samples/python"))
+SAMPLES_CPP_DIR = os.environ.get("SAMPLES_CPP_DIR", os.getcwd())
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--cleanup-cache", action="store_true", default=False, help="Cleanup cache after tests"
```

Comment:
Why is it a classic cmd arg? `SAMPLES_CPP_DIR` is the env var. I'd keep all genai defined parameters consistent.

Describe it and `OV_CACHE` in .md.

Reply:
would you like to use env or command lines arguments only?

Reply:
I like env vars more

Reply:
changed